packages/web-react/src/components/Radio/demo/RadioDefault.tsx

```diff
@@ -3,9 +3,9 @@ import Radio from '../Radio';
 
 const RadioDefault = () => (
   <>
-    <Radio id="radio-default" label="Radio Label" name="radioDefault" />
+    <Radio id="radio-default" label="Radio Label" name="radioDefault" onChange={() => {}} />
```

Comment:
Why is it necessary to apply the `onChange` prop on this component? And shouldn't it be part of the README and component's interface?